plugins/nextcloud-rule-exclusions-before.conf

```diff
@@ -554,12 +556,17 @@ SecRule REQUEST_FILENAME "@rx /apps/text/(?:public/)?(?:session/(?:sync|close|pu
     nolog,\
     ver:'nextcloud-rule-exclusions-plugin/1.0.0',\
     chain"
-    SecRule ARGS:json.sessionToken "@rx ^(?i)[a-z0-9+/]+$" \
+    SecRule ARGS:sessionToken|ARGS:json.sessionToken "@rx ^(?i)[a-z0-9+/]+$" \
```

Comment:
As this regex is technically correct i would move `(?i)` before `^` as it's probably meant to apply for the whole regex..